mycore-solr/src/main/resources/META-INF/resources/search/search-expert.xed

```diff
@@ -121,6 +121,19 @@ form .optional.expanded .fieldset{
                 </h3>
               </div>
               <div class="panel-body">
+                <xed:bind xpath="param[@name='core']" default="main">
+                  <div class="form-group">
+                    <label class="col-md-{$label-width} control-label" for="core">
+                      core
+                    </label>
+                    <div class="col-md-{$input-width}">
+                      <select title="SOLR core (alias)." class="form-control" id="core">
+                        <option>main</option>
+                        <option>classification</option>
+                      </select>
+                    </div>
+                  </div>
+                </xed:bind>
```

Comment:
I'm not sure if this is the best solution. It is user friendly, but filters flexibility. We can specify more core types (not only classification and core). If so, we have to add this here ...

Reply:
this will not work for an undefined set of core aliases but only for a fixed set. a 'real' expert has to change the URL parameter to any other core. No field names and types will be fetched in either case.

Reply:
another possibility would be to replace the select box with an input field that defaults to 'main'

Reply:
This sounds good for me. In the most use cases "main" will be used I think.